video.go

```diff
@@ -37,6 +37,24 @@ type VideoOutputs struct {
 	MP4  string
 }
 
+// LoopOffsetOptions is a set of options for rendering loop offset
+type LoopOffsetOptions struct {
+	frames     int
```

Comment:
I think we can actually remove the # of frames options for a few reasons:

1. It's nice to have only one way of doing things (percentage loop offsets) it will make documentation and code nicer.
2. I doubt anyone will need to get so precise as to specify the frame count (percentages are better) since they won't know the frame count or frame number in advanced anyway.
3. Even if people did know the exact frame number they wanted VHS isn't deterministic so these frame counts can change based on the machine running it so it would be more repeatable since percentages would work across framerates (and work well with the `Set Framerate` setting.

Reply:
I agree. I myself ended up using percentages for testing. 

Does using `VHS.Options.LoopOffset` to store percentage sound good? I will get rid of `LoopOffsetOptions` struct altogether.

Reply:
Yep, that's perfect!

Reply:
Should we keep `%` token for semantics? 
```elixir
Set LoopOffset 20.45 # implicit 
# OR
Set LoopOffset 20.45%
```

Reply:
I went with keeping it optional.